### PR TITLE
Feat - Added Navigation Component

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
#### Change Log
Added Simple navigation component

### CODE REVIEW

**Review by shrushti2000 — COMMENTED** [review]
Looks good to me overall. Just a suggestion , you can add meta tags and also leave space between each CSS declaration.

File: css/components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  color: var(--text--primary--color);
+  font-size: 1.5rem;
+  background-color: var(--secondary--color);
+}
+.nav-left-sec {
+  gap: 1rem;
+}
+.nav-right-sec {
+  margin-left: auto;
+  margin-right: 0;
```
Comment (viru2001) [inline css/components/navigation.css]: you can remove `margin-right:0;`. since by default it is 0

**Review by viru2001 — COMMENTED** [review]
everything looks good, I gave small suggestion.

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @viru2001 , @shrushti2000 for reviewing

**Pull request merged**